.github/workflows/update-spellcheck-binaries.yml

```diff
@@ -0,0 +1,71 @@
+name: Update node-spellchecker binaries
+on:
+  push:
+    paths:
+      - 'lib/bin/node-spellchecker/**'
+jobs:
+  windows-binaries:
+    name: Windows Binaries
+    runs-on: windows-2019
```

Comment:
If it were only for Github Action automation, the setup could be a single matrix build configuration over platforms and nodejs versions, running the commands in the shell scripts directly from here. 

But it would be an even bigger change, would be slower because checkout, install etc would happen separately for every binary and for local builds of the binaries the script would still be needed.  Also, matrix builds run in parallel and would interfere when trying to commit the built artifacts.